doc/nrf/shortcuts.txt

```diff
@@ -86,3 +86,5 @@
 .. |vsc_extension_instructions| replace:: The `nRF Connect for Visual Studio Code`_ extension is a complete IDE for developing applications for nRF91, nRF53 and nRF52 Series devices. This includes an interface to the compiler and linker, an RTOS-aware debugger, a seamless interface to the nRF Connect SDK, and a serial terminal.
 
 .. |VSC| replace:: nRF Connect for Visual Studio Code
+
+.. |thingy53_sample_note| replace:: If you use this application with Thingy:53, it complies with Thingy:53 application guide and enables additional features. See :ref:`thingy53_app_guide` for details.
```

Comment:
This description still mentions `complies with`.

Reply:
Applied change requested by @wiba-nordic. Now I rephrased it again. Please verify if it's ok

Reply:
Oh, seems one of my suggestions has reverted to an unedited version I first made. For this line, I intended to suggest:
```suggestion
.. |thingy53_sample_note| replace:: If you use this application with Thingy:53, it enables additional features. See :ref:`thingy53_app_guide` for details.
```